client/my-sites/email/gsuite-purchase-features/index.jsx

```diff
@@ -124,7 +129,7 @@ const learnMoreClick = domainName =>
 
 GSuitePurchaseFeatures.propTypes = {
 	domainName: PropTypes.string.isRequired,
-	productSlug: PropTypes.string.isRequired,
+	productSlug: PropTypes.string,
```

Comment:
Do we not need a slug?

Reply:
nope, not passing in a productSlug gets the plan-agnostic "30GB or unlimited storage" mentioned above